test/libraries/BytesArrayUtil.t.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Test, stdError} from "forge-std/Test.sol";
+import {BytesArrayUtil} from "../../src/libraries/BytesArrayUtil.sol";
+
+import "../../src/config/constants.sol";
+import "../../src/config/errors.sol";
+import "../../src/config/types.sol";
+
+/**
+ * Basic tests
+ */
+contract BytesArrayUtilTest is Test {}
```

Comment:
Are there supposed to be tests here?

Reply:
Added in tests for `append` and `indexOf`